kod/object/passive/itematt/weapatt/waspellt.kod

```diff
@@ -36,6 +36,9 @@ constants:
 
    include blakston.khd
 
+   ONE_HOUR_IN_MILLISECONDS = 3600000
+   ONE_DAY_IN_MILLISECONDS = 86400000
```

Comment:
Same feedback here... 60 * 60 * 24 * 1000 is clearer.  No counting zeros to see if it's correct.  (And no performance penalty, as the compiler will simplify the expression.)

Reply:
👍🏻